src/gui_common/thrive_theme.tres

```diff
@@ -20,7 +20,7 @@
 [ext_resource path="res://assets/textures/gui/bevel/checkAhover.png" type="Texture" id=18]
 [ext_resource path="res://assets/textures/gui/bevel/checkBclick.png" type="Texture" id=19]
 [ext_resource path="res://assets/textures/gui/bevel/checkBhover.png" type="Texture" id=20]
-[ext_resource path="res://src/gui_common/fonts/Lato-Bold-AlmostSmall.tres" type="DynamicFont" id=21]
+[ext_resource path="res://src/gui_common/fonts/Lato-Bold-Small.tres" type="DynamicFont" id=21]
```

Comment:
What's this about?

Reply:
Tweaking the bold font for our custom rich text label to have consistent size as the other fonts.